docs/cloud/cloud-quickstart.md

```diff
@@ -108,7 +108,7 @@ If you choose to log in via the browser, Prefect opens a new tab in your default
 
 ## Run a flow with Prefect Cloud
 
-Okay, you're all set to run a local flow with Prefect Cloud. Notice that everything works just like [running local flows](/tutorial/first-steps/). However, because you logged into Prefect Cloud locally, your local flow runs show up in Prefect Cloud!
+Okay, you're all set to run a local flow with Prefect Cloud. Notice that everything works just like [running local flows](/tutorial/flows/). However, because you logged into Prefect Cloud locally, your local flow runs show up in Prefect Cloud!
```

Comment:
```suggestion
Okay, you're all set to run a flow locally, orchestrated with Prefect Cloud. Notice that everything works just like [running local flows](/tutorial/flows/). However, because you logged into Prefect Cloud, your local flow runs show up in Prefect Cloud!
```